packages/caraml-auth-google/requirements.dev.txt

```diff
@@ -0,0 +1,6 @@
+black==22.6.0
+setuptools>=21.0.0
+wheel
+twine
+pytest
```

Comment:
We don't have any test at the moment :)

But can we add it in? https://github.com/apache/airflow/blob/9ab1a6a3e70b32a3cddddf0adede5d2f3f7e29ea/tests/providers/google/common/utils/test_id_token_credentials.py

Reply:
Okay I've added the test suite as well as a makefile, and a CI job to run it :)